docs/sources/configure/_index.md

```diff
@@ -2029,6 +2029,10 @@ boltdb_shipper:
     [grpc_client_config: <grpc_client>]
 
     # Hostname or IP of the Index Gateway gRPC server running in simple mode.
+    # Can also be prefixed with dns+, dnssrv+, or dnssrvnoa+ to resolve a DNS A
+    # record with multiple IP's, a DNS SRV record with a followup A record
+    # lookup, or a DNS SRV record without a followup A record lookup,
+    # respectively.
```

Comment:
```suggestion
    # Can also be prefixed with dns+, dnssrv+, or dnssrvnoa+ to resolve a DNS, a
    # record with multiple IP's, a DNS SRV record with a followup, a record
    # lookup, or a DNS SRV record without a followup, a record lookup,
    # respectively.
```

Reply:
actually these are specifically referring to the [A record type for DNS](https://www.cloudflare.com/learning/dns/dns-records/dns-a-record/)